catalog/CSIC/CSIC_SPEI_2_8.jsonnet

```diff
@@ -0,0 +1,469 @@
+	        '(SPEI) where precipitation and evapotranspiration data was  ' +
+	        'accumulated over the previous 38 months',
+      },
+      {
+        name: 'SPEI_39_month',
+        description: 'Standardized Precipitation-Evapotranspiration Index ' +
+	        '(SPEI) where precipitation and evapotranspiration data was  ' +
+	        'accumulated over the previous 39 months',
+      },
+      {
+        name: 'SPEI_40_month',
+        description: 'Standardized Precipitation-Evapotranspiration Index ' +
+	        '(SPEI) where precipitation and evapotranspiration data was  ' +
+	        'accumulated over the previous 40 months',
+      },
+      {
+        name: 'SPEI_41_month',
+        description: 'Standardized Precipitation-Evapotranspiration Index ' +
+	        '(SPEI) where precipitation and evapotranspiration data was  ' +
+	        'accumulated over the previous 41 months',
+      },
+      {
+        name: 'SPEI_42_month',
+        description: 'Standardized Precipitation-Evapotranspiration Index ' +
+	        '(SPEI) where precipitation and evapotranspiration data was  ' +
+	        'accumulated over the previous 42 months',
+      },
+      {
+        name: 'SPEI_43_month',
+        description: 'Standardized Precipitation-Evapotranspiration Index ' +
+	        '(SPEI) where precipitation and evapotranspiration data was  ' +
+	        'accumulated over the previous 43 months',
+      },
+      {
+        name: 'SPEI_44_month',
+        description: 'Standardized Precipitation-Evapotranspiration Index ' +
+	        '(SPEI) where precipitation and evapotranspiration data was  ' +
+	        'accumulated over the previous 44 months',
+      },
+      {
+        name: 'SPEI_45_month',
+        description: 'Standardized Precipitation-Evapotranspiration Index ' +
+	        '(SPEI) where precipitation and evapotranspiration data was  ' +
+	        'accumulated over the previous 45 months',
+      },
+      {
+        name: 'SPEI_46_month',
+        description: 'Standardized Precipitation-Evapotranspiration Index ' +
+	        '(SPEI) where precipitation and evapotranspiration data was  ' +
+	        'accumulated over the previous 46 months',
+      },
+      {
+        name: 'SPEI_47_month',
+        description: 'Standardized Precipitation-Evapotranspiration Index ' +
+	        '(SPEI) where precipitation and evapotranspiration data was  ' +
+	        'accumulated over the previous 47 months',
+      },
+      {
+        name: 'SPEI_48_month',
+        description: 'Standardized Precipitation-Evapotranspiration Index ' +
+	        '(SPEI) where precipitation and evapotranspiration data was  ' +
+	        'accumulated over the previous 48 months',
+      },
+    ],
+    'gee:visualizations': [
+      {
+        display_name: 'SPEI',
+        lookat: {lon: -3.75, lat: 40.47, zoom: 4},
+        image_visualization: {
+          band_vis: {
+            min: -2.33,
```

Comment:
This error in the checker is an issue in the Jsonnet and a bug in the checker (it should not have crashed)

```
  File "/home/runner/work/earthengine-catalog/earthengine-catalog/bazel-bin/checker/ee_stac_check.runfiles/__main__/checker/ee_stac_check.py", line 57, in find_issues
    yield from node.run_checks(a_node, checks)
  File "/home/runner/work/earthengine-catalog/earthengine-catalog/bazel-bin/checker/ee_stac_check.runfiles/__main__/checker/node/__init__.py", line 106, in run_checks
    yield from check.run(node)
  File "/home/runner/work/earthengine-catalog/earthengine-catalog/bazel-bin/checker/ee_stac_check.runfiles/__main__/checker/node/visualizations_image.py", line 187, in run
    min_list_size = len(min_list)
TypeError: object of type 'float' has no len()
Error: Process completed with exit code 1.
```

The problem is here:

```
          band_vis: {
            min: -2.33,  // <--
            max:  2.33, // <--
            palette: [
```

In the Jsonnet, the min and max fields have to be lists.

```
          band_vis: {
            min: [-2.33],
            max: [2.33],
            palette: [
```

Reply:
Thank you, I have already fixed the issue.